### DISCUSSION TITLE
Add breadcrumb navigation?

### DISCUSSION THREAD

**Discussion by jdillard** [pradyunsg/furo#139] [Ideas] [answered]
I'm a fan a breadcrumb navigation, would that be something that would be a welcome addition to the theme?

**Comment by pradyunsg** [comment]
If you can figure out a good way to include it into the design of the site, yes.

Something like the breadcrumbs in https://pmarsceill.github.io/just-the-docs/docs/utilities/layout/ might be a good addition. :)

**Reply by feorlen** [reply]
I had feedback from a reader suggesting we have breadcrumbs too, have there been any recent thoughts about how someone might accomplish it? 

Btw, that link to `just-the-docs` no longer exists. The Wayback Machine has it, however.
<img width="538" alt="Screen Shot 2022-03-22 at 17 07 28" src="https://user-images.githubusercontent.com/95028/159596742-5d210352-d72a-4bb6-a2ad-f536cacffe11.png">

**Reply by pradyunsg** [reply]
Yea, the site moved to https://just-the-docs.github.io/just-the-docs/.

I have an implementation for this, in `sphinx-basic-ng` (https://sphinx-basic-ng.readthedocs.io/en/latest/usage/components/breadcrumbs/) but haven't had the time to port Furo over to using that theme as the base yet.

**Reply by Eric-Arellano** [reply]
Hey Pradyun, we need to add breadcrumbs to qiskit_sphinx_theme as part of our Furo migration. I would love to contribute that code upstream to Furo. 

We quite like the design from Just the Docs! So my PR would be based on that. How would you feel about me submitting that patch?

**Reply by pradyunsg** [reply, marked as answer]
File a PR! :)

**Reply by pradyunsg** [reply]
I've opened https://github.com/pradyunsg/furo/issues/680 for this.
